src/main/java/ch/njol/skript/expressions/ExprResonatingTime.java

```diff
@@ -0,0 +1,68 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.RequiredPlugins;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.skript.util.Timespan;
+import org.bukkit.block.Bell;
+import org.bukkit.block.Block;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Resonating Time")
+@Description({
+	"Returns the resonating time of a bell.",
+	"A bell will start resonating five game ticks after being rung, and will continue to resonate for 40 game ticks."
+})
+@Examples("broadcast \"The bell has been ringing for %ringing time of target block%\"")
+@RequiredPlugins("Minecraft 1.19.4+")
+@Since("INSERT VERSION")
+public class ExprResonatingTime extends SimplePropertyExpression<Block, Timespan> {
+
+	static {
+		if (Skript.classExists("org.bukkit.block.Bell") && Skript.methodExists(Bell.class, "getResonatingTicks")) {
+			register(ExprResonatingTime.class, Timespan.class, "resonat(e|ing) time", "block");
+		}
```

Comment:
If the method is guaranteed to be in the class no matter the version. The methodExists should be removed to reduce reflection time taken on initialization.
```suggestion
		if (Skript.classExists("org.bukkit.block.Bell") && Skript.methodExists(Bell.class, "getResonatingTicks"))
			register(ExprResonatingTime.class, Timespan.class, "resonat(e|ing) time", "block");
```

Reply:
Sadly, it's not :( Early versions had Bell but not the relevant methods

Reply:
![image](https://github.com/SkriptLang/Skript/assets/28607612/fabd444c-9cb0-43c5-86e0-eb0bb0a43b4d)